src/dos/dos_programs.cpp

```diff
@@ -496,6 +497,43 @@ void DOS_SetupPrograms(void)
 	MSG_Add("PROGRAM_KEYB_LAYOUTNOTFOUND","No layout in %s for codepage %i\n");
 	MSG_Add("PROGRAM_KEYB_INVCPFILE","None or invalid codepage file for layout %s\n\n");
 
+	MSG_Add("PROGRAM_SERIAL_HELP",
+	        "Allows on-the-fly serial port reconfiguration.\n"
+	        "\n"
+	        "Usage:\n"
+	        "  SERIAL [[/l | /list] | [port#] [type] {options}]\n"
+	        "\n"
+	        "Where:\n"
+	        "  /l or /list displays the current serial port configuration.\n"
+	        "  [port#] [type] {options} specifies a new configuration.\n"
+	        "\n"
+	        "Port # (required):\n"
+	        "  1 through %d\n"
+	        "\n"
+	        "Type (required):\n"
+	        "\n"
+	        "  Can be one of:  DISABLED, DUMMY, DIRECTSERIAL, MODEM, or NULLMODEM."
+	        "\n"
+	        "Options (mostly optional, depends on port type):\n"
+	        "  For DISABLED: none.\n"
+	        "  For DUMMY: IRQ\n"
+	        "  For DIRECTSERIAL: IRQ, REALPORT (required), RXDELAY\n"
+	        "  For MODEM: IRQ, LISTENPORT, SOCK\n"
+	        "  For NULLMODEM: IRQ, SERVER, RXDELAY, TXDELAY, TELNET, USEDTR,\n"
+	        "                 TRANSPARENT, PORT, INHSOCKET, SOCK\n"
+	        "\n"
+	        "Examples:\n"
+	        "  SERIAL /l\n"
+	        "  SERIAL 1 DISABLED\n"
+	        "  SERIAL 2 MODEM LISTENPORT:5000 SOCK:1\n"
+	        "  SERIAL 1 DIRECTSERIAL REALPORT:COM1\n"
+	        "  SERIAL 1 DIRECTSERIAL REALPORT:tty50\n");
```

Comment:
This really should be somehow shortened. Currently it doesn't fit into default 80x25 window.

Reply:
Can you take a screenshot of it in a 80x25 window so we have a better idea how to make it more consise?

Reply:
Sure.
![Screenshot_20211215_091825](https://user-images.githubusercontent.com/11149380/146495838-3347d043-44c2-47d7-a4c2-869ee6638927.png)

Reply:
Can you make one in 80x43 too so we can see the whole text.

Reply:
That's `MODE 80,43` with FreeDOS.

Reply:
![Screenshot_20211217_105747](https://user-images.githubusercontent.com/11149380/146496610-8a4ba5aa-caee-4bc3-8f0e-cf39151157f4.png)

Reply:
Usage: <eliminate \n here

Port # (required): <eliminate \n here>

Type (required): <eliminate \n here>

Examples: <eliminate \n here>

Eliminate line: SERIAL 1 DIRECTSERIAL REALPORT:tty50

Reply:
![2021-12-17_08-51](https://user-images.githubusercontent.com/1557255/146580167-e88aa521-6d71-47d9-9e85-c1a039950d08.png)